library/core/src/cell.rs

```diff
@@ -1036,7 +1035,7 @@ impl<T: Default> RefCell<T> {
     /// assert_eq!(five, 5);
     /// assert_eq!(c.into_inner(), 0);
     /// ```
-    #[unstable(feature = "refcell_take", issue = "71395")]
+    #[stable(feature = "refcell_take", since = "1.49.0")]
```

Comment:
The version has been bumped to 1.50.0: https://github.com/rust-lang/rust/pull/79014
```suggestion
    #[stable(feature = "refcell_take", since = "1.50.0")]
```

Reply:
Thanks! I forced pushed with `1.50.0`.